.github/workflows/build.yaml

```diff
@@ -7,22 +7,52 @@ name: Buid iOS app
 on:
   workflow_call:
 
+env:
+  spm_dir: './SPM'
+  cocoapods_dir: './CocoaPods'
+  platform: 'iOS Simulator'
+  device_name: 'iPhone 14'
+  ios_version: '16.2'
```

Comment:
How do we choose the iOS version here?

Reply:
Maybe it shouldn't be iOS 17?

Reply:
Currently, the SDK is officially supported by Xcode 13 and 14. With the official support of Xcode 15, it will be updated to target iOS 17 as well.